src/Modules/tsqlScheduler/tsqlScheduler.psm1

```diff
@@ -6,18 +6,32 @@ Function Install-SchedulerSolution
         ,[string] $database
         ,[boolean] $agMode = $false
         ,[string] $availabilityGroup
+        ,[boolean] $versionBump = $false
     )
 
     $deployMode = if($agMode){"IsAGMode"}else{"IsStandaloneMode"}
     $compileInclude = Import-Csv ..\deploy\compileInclude.csv
 
-    $files += $compileInclude | Where-Object { $_."$deployMode" -match $true } 
+    if($versionBump){
+        $files += $compileInclude | Where-Object { 
+            ($_."$deployMode" -match $true) -and
+            (($_.fileName).StartsWith(".\Tables") -eq $false) # ignore stateful data objs on version bump
+        }
+    }else{
+        $files += $compileInclude | Where-Object { $_."$deployMode" -match $true } 
+    }
+
+    Write-Verbose ">>>>>>> $server"
```

Comment:
I think this is useful for interactive debugging but we could probably be more concise in the future with verbose output, e.g. [Time][FunctionName]Doing operation on $server - so that each line of verbose has value without context or surrounding.  Probably be able to collapse multiple lines together and get rid of the formatting too (though I wouldn't rework it now).

Write-Debug is always available for messages we want to see when debugging/building.